docs/tutorials/alert-remediation.rst

```diff
@@ -13,23 +13,40 @@ Add the following to your :ref:`customPlaybooks<customPlaybooks>`:
 .. code-block:: yaml
 
     customPlaybooks:
+    # Change the following line according to your needs
     - triggers:
-      - on_prometheus_alert:
-          alert_name: SomeAlert
+        - on_prometheus_alert:
+            alert_name: TestAlert
       actions:
-      - alert_handling_job:
-          command:
-          - "echo"
-          - "do something to fix it"
-          image: "a_docker_image"
+        - alert_handling_job:
+            # you can access information from the alert by environment variables
+            command:
+              - sh
+              - -c
+              - "echo '$ALERT_NAME fired... Now dumping all available environment variables, which include alert metadata and labels' && env && sleep 60"
```

Comment:
@pavangudiwada can you add to the example something explicit showing how to print one of the labels from the alert?